common/app/views/fragments/amp/onwardTemplateAmp.scala.html

```diff
@@ -10,49 +10,51 @@
                     <span>{{displayName}}</span>
                 </div>
                 {{#content}}
-                    <div class="fc-item {{toneClass}}">
-                        <div class="fc-item__image-container">
-                            <amp-img src="{{thumbnail}}" layout="fixed" width=126 height=75></amp-img>
-                        </div>
-                        <div class="fc-item__content">
-                            <div class="fc-item__header">
-                                <h2 class="fc-item__title">
-                                    <span>
-                                        {{#isVideo}}
-                                            @fragments.inlineSvg("video-icon", "icon")
-                                        {{/isVideo}}
-                                        {{#isGallery}}
-                                            @fragments.inlineSvg("camera", "icon")
-                                        {{/isGallery}}
-                                        {{#isAudio}}
-                                            @fragments.inlineSvg("volume-high", "icon")
-                                        {{/isAudio}}
-                                        {{#isComment}}
-                                            @fragments.inlineSvg("quote", "icon")
-                                        {{/isComment}}
-                                    </span>
-                                    {{headline}}
-                                </h2>
-                                {{#isComment}}
-                                <div class="fc-item__byline">
-                                    {{byline}}
-                                </div>
-                                {{/isComment}}
+                    {{#headline}} @* Don't show if headline is empty*@
```

Comment:
diff is confusing because of intendation but the change is simple:
- Check if headline is non empty before to display a row